pkg/core/native/native_nep5.go

```diff
@@ -146,7 +148,16 @@ func (c *nep5TokenNative) transfer(ic *interop.Context, from, to util.Uint160, a
 	if amount.Sign() == -1 {
 		return errors.New("negative amount")
 	}
-
+	ok, err := runtime.CheckHashedWitness(ic, nep5ScriptHash{
+		callingScriptHash: c.Hash,
+		entryScriptHash:   c.Hash,
+		currentScriptHash: c.Hash,
+	}, from)
+	if err != nil {
+		return err
+	} else if !ok {
+		return errors.New("invalid signature")
+	}
```

Comment:
It's a separate fix for a separate problem.

Reply:
Moved this to separate commit.